synapse/federation/transport/server.py

```diff
@@ -703,6 +703,25 @@ def on_DELETE(self, origin, content, query, group_id, room_id):
         defer.returnValue((200, new_content))
 
 
+class FederationGroupsAddRoomsConfigServlet(BaseFederationServlet):
+    """Update room config in group
+    """
+    PATH = "/groups/(?P<group_id>[^/]*)/room/(?P<room_id>[^/]*)"
+    "/config/(?P<config_key>[^/]*)$"
```

Comment:
Is this supposed to be unindented? I'm guessing yes, I just haven't seen this before

Reply:
Woops. I need to add brackets...